coralproject/experiment-playground
# Adding share buttons and about link.

## What does this PR do?

Adds about button and share buttons.
## How do I test this PR?
- Click the about button in the header and footer, you should see a modal with information about the Playground.
- Click the Facebook and Twitter share buttons. You should be prompted to share.
